Submissions/mluevanos.json

```diff
@@ -1,9 +1,9 @@
-{
+  {
     "name": "Michael Luevanos",
-    "img": "https://photos.app.goo.gl/mE8U6jyowMYy5sMV7",
-    "email":"",
-  "links": {
-      "website": "MichaelLuevanos.com",
+    "img": "https://avatars.githubusercontent.com/u/54725055?v=4",
```

Comment:
Instead of using 
> https://avatars.githubusercontent.com/u/54725055?v=4
try using 
> https://github.com/mluevanos.png

whenever you will change your photo it will update it.